llumnix/llumlet/llumlet.py

```diff
@@ -157,8 +158,14 @@ async def migrate_out(self, dst_instance_name: str, num_requests: int) -> List[s
                 logger.info("{}->{} migrate done, migrate request {}, status:{}, len:{} blocks, cost:{} ms" \
                     .format(self.instance_id, dst_instance_id, migrated_request_list, status, \
                     sum(migrate_out_request.stage_num_blocks_list), (t1 - t0)*1000))
-        except ray.exceptions.RayActorError:
-            logger.info("[migrate_out] instance {} is dead".format(dst_instance_name[len("instance_"):]))
+        # pylint: disable=broad-except
+        except Exception as e:
+            if migrate_out_request:
+                self.migration_scheduler.finish_migration(migrate_out_request.request_id)
+
+            logger.error("[migrate_out] src instance {}, dst instance {}, meet error: {}"
```

Comment:
don't change exception in this pr, leave it for me in a seperate pr.

Reply:
ok